pkg/api/jobs/jobs_test.go

```diff
@@ -0,0 +1,103 @@
+package jobs
+
+import (
+	"encoding/json"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestReturnAllJobs(t *testing.T) {
```

Comment:
Are we using test data for jobs, I don't see it.

Reply:
Check the Init in jobs.go

Reply:
yeah but that is not test data.  I guess this is ok for an initial unit test.